src/migrations/r-migration-commands/m128_129_delete_whitelabel_add_show_splash.ts

```diff
@@ -0,0 +1,38 @@
+import { RecordNode, r, rtp, RT } from "../../r";
+import { IOrder } from "../IOrder";
+
+/**
+ * Renames project property
+ * whitelabel -> show_powered_by_gmetri + show_splash_screen
+ * 
+ * Earlier, whitelabel's default value was false. So if whitelabel was:
+ * true - show_powered_by_gmetri and show_splash_screen: false
+ * false - show_powered_by_gmetri and show_splash_screen: true
+ * undefined/null - show_powered_by_gmetri and show_splash_screen: can remain undefined.
+ * (whitelabel default: false, show_powered_by_gmetri and show_splash_screen default: true)
+ */
+
+class Migration implements IOrder {
+  execute (projectJson: unknown) {
+    const pJson = projectJson as RecordNode<RT.project>;
+    const projectF = r.record(pJson);
+
+    projectF.changePropertyName("whitelabel", "show_powered_by_gmetri");
+
+    //Change property values
+    if(projectF.get(rtp.project.show_powered_by_gmetri) === true) {
```

Comment:
extract this out into a const. 

`const showPoweredByGMetri = projectF.get(rtp.project.show_powered_by_gmetri)`